modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+The {productname} 6.1 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.1.0 introduces the following enhancements:
+
+
+==== A new event log API
+
+A new API, `tinymce.activeEditor.plugins.tinycomments.getEventLog()` is now available.
+
+As its name suggests, it returns a log of all Comment-related events, including:
+
+* when a new comment is added (and who added it).
+
+* when a coment is edited (and who edited it).
+
+* when a reply to a comment is added (and who added the reply).
+
+* when a comment is resolved (and who resolved the comment).
+
+* when a comment is deleted (and who deleted the comment).
+
+The new API also supports an `after` option.
+
+[source,js]
+////
+tinymce.activeEditor.plugins.tinycomments.getEventLog(
+	{after: '<iso-8601-format-time-stamp>'}
+	)
+////
+
+restricts the returned list to Comment events after the time-stamp date.
+
+Underlying this new API is a new `CommentChange` event, which gets fired when any modification is made to Comments data.
+
+
+==== Avatars can now be displayed next to usernames in comments
+
+The default icon presented next to user names in comments can now be replaced by a custom avatar.
+
+In embedded mode, the avatar is supplied by the `tinycomments_author_avatar` option:
+
+[source,js]
+----
+    plugins: [
+        "tinycomments",
+    ],
+	tinycomments_mode: "embedded",
+	tinycomments_author: "testuser",
+	tinycomments_author_name: "Test User",
+	tinycomments_author_avatar: "https://example.com/image.ext",
+----
+
+In callback mode, the avatar is supplied by the `avatar` option of `setConverstation()`:
+
+[source,js]
+----
+const avatarUrl = "https://example.com/image.ext";
+
+/*
+
+much non-avatar-related configuration here
+
+*/
+
+      setConversation(conversationUid, [
+        {
+          author,
+		  avatar: avatarUrl,
+          createdAt,
+          modifiedAt: createdAt,
+          content,
+          uid: conversationUid /* first comment has same uid as conversation */,
+        },
+      ]);
+----
+
+In both modes the avatar:
+
+* must be 36px by 36px.
+
+* can be any filetype able to be wrapped in an `<img>` element.
+
+* will present as the browser’s default ‘image not found’ icon if the avatar resource is not available at the nominated URL.
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, showing avatars next to the user-name in comments requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+==== Show comments sidebar on editor initialization
+
+It is now possible to initialize a {productname} editor with a comment sidebar automatically visible and available but with focus still in the main editor.
+
+To add a comment sidebar add `sidebar_show: 'showcomments'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",
+	sidebar_show: 'showcomments',
+});
+----
```

Comment:
Again this snippet doesn't follow the style guide, so this should correct it:
```suggestion
----
tinymce.init({
  selector: "textarea", // change this value according to your HTML
  plugins: 'tinycomments',
  sidebar_show: 'showcomments',
});
----
```

Reply:
Added the corrected code-sample locally (and then pushed to the branch). I’d already done so many local edits at this point the prospect of changing the file from two directions (so to speak) and merging was… unpleasant.